cpython-unix/build.py

```diff
@@ -255,10 +256,9 @@ def simple_build(
         build_env.copy_file(SUPPORT / ("build-%s.sh" % entry))
 
         env = {
-            "%s_VERSION"
-            % entry.upper()
-            .replace("-", "_")
-            .replace(".", "_"): DOWNLOADS[entry]["version"],
+            "%s_VERSION" % entry.upper().replace("-", "_").replace(".", "_"): DOWNLOADS[
+                entry
+            ]["version"],
```

Comment:
Sorry, some of these look like format-on-save. I can revert.

Reply:
I thought I had black formatting on for all the Python files in this repo. It is possible I haven't picked up a modern version of black yet.

I'd welcome PRs to integrate ruff :)

Reply:
I'll open a pull request to add Ruff :D